src/components/WritePost.tsx

```diff
@@ -84,12 +85,13 @@ const WritePost: React.FC = () => {
             </IonItem>
             <IonItem>
                 <IonLabel>Content</IonLabel>
-                <IonInput
+                <IonTextarea
                     spellcheck={true}
-                    type="text"
+                    autoGrow={true}
+                    inputMode="text"
                     value={content}
                     onIonChange={(e: CustomEvent<InputChangeEventDetail>) => setContent(e.detail.value ?? '')}
-                ></IonInput>
+                ></IonTextarea>
```

Comment:
Very small stylistic choice but you could change this line to be ` />` instead of `></IonTextarea>`. Looks a little cleaner since there's no children components

Reply:
Ok I'll fix that